DesignSafe-CI/portal
# feat: DES-2903 add id to app listing headings

## Overview

Add an `id` attribute to headings of App Listing CMS plugin.

## Status


CMS on Prod and Next is ready to use this also, because I updated their category pages to redirect to anchors e.g. `/use-designsafe/tools-applications#simulation`.

## Related

* [DES-2903](https://tacc-main.atlassian.net/browse/DES-2903)
* can fix category breadcrumb in https://github.com/DesignSafe-CI/portal/pull/1268#discussion_r1643316018

## Changes

- **add** `id` attribute

## Testing

0. Have at least one entry in "App Categories".
1. Create/Load a page in the CMS.
2. Add/View an "App Listing" plugin instance.
3. Verify the `<h2>` heading has an `id` attribute.

## UI

<img width="880" alt="Screenshot 2024-06-25 at 5 02 28 PM" src="https://github.com/DesignSafe-CI/portal/assets/62723358/5854863c-eb8b-4ae0-b39d-e5725290a4bf">
